Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -50,6 +50,8 @@ public class SiteExtensionManager : ISiteExtensionManager
         private readonly IContinuousJobsManager _continuousJobManager;
         private readonly ITriggeredJobsManager _triggeredJobManager;
 
+        private static readonly string _toBeDeletedDirectoryPath = Path.Combine(System.Environment.ExpandEnvironmentVariables(@"%HOME%\data\SiteExtensions"), "tobedeleted");
```

Comment:
This is _toBeDeleted, no?  meaning it is a place holder till deletion.   why same or different machine is relevant?